com.github.glitterware.Passy.yml

```diff
@@ -0,0 +1,39 @@
+app-id: com.github.glitterware.Passy
```

Comment:
1. I said `io.github`, not `com.github`.

Reply:
Any reason for than? I would prefer `com.github` over `io.github`, however rules are rules and I'm ready to follow once explained. I used `com.github.LongSoft.UEFITool` as an example.

Reply:
it was not enforced back then, it is now.

Also in the doc:
https://docs.flatpak.org/en/latest/conventions.html#application-ids